clients/da-vinci-client/src/test/java/com/linkedin/davinci/store/rocksdb/RocksDBStorageEngineTest.java

```diff
@@ -67,6 +69,24 @@ public void cleanUp() throws Exception {
     storageService.stop();
   }
 
+  @AfterMethod
+  public void testCounter() {
+    this.testCount++;
+  }
+
+  /**
+   * Some tests require a reset if other tests have run before them, as they are sensitive to contamination.
+   *
+   * Alternatively, we could make {@link #setUp()} have {@link org.testng.annotations.BeforeMethod} and
+   * {@link #cleanUp()} have {@link AfterMethod}, though that makes the class take longer than the current approach.
```

Comment:
nit: can we do that in this PR? I prefer this way as it's cleaner and don't get people confused when they touch this file in the future. However, if it takes a lot more effort than expected, I suppose we are okay with this too.

Reply:
This is how I did at first, but it made the test take longer to run, so I iterated until the current solution... effort-wise it's basically a no-op.